### ISSUE TITLE
Warning in IconActivity.vue:2:62 while compiling/running the server

### ISSUE THREAD

**Issue by LaurentBonnaud** [c2corg/c2c_ui#337]
Hi,

here is the problem:

```
$ npm run serve
[...]
 WARNING  Compiled with 1 warnings                                                                           3:39:53 PM

Module Warning (from ./node_modules/eslint-loader/index.js):
warning: Expected a space before '/>', but not found (vue/html-closing-bracket-spacing) at src/components/generics/icons/IconActivity.vue:2:62:
  1 | <template>
> 2 |   <fa-icon :icon="['activity', activity]" :class="'no-print'"/>
    |                                                              ^
  3 | </template>
  4 |
  5 | <script>


1 warning found.
1 warning potentially fixable with the `--fix` option.
```

**Comment by cbeauchesne** [comment]
Yes seen. @brunobesson had run `npm run lint` on it's branch : https://github.com/c2corg/c2c_ui/commit/3100525eca60e51718b6441d91243f1e5b5fab82

It will be fixed when he'll merge it.

**Comment by brunobesson** [comment]
Fixed by #339
